packages/discord-bot/src/discord/commands/setEthAddress.ts

```diff
@@ -45,10 +45,24 @@ export abstract class SetEthAddress {
       return;
     }
 
+    const account = manager.ledger.account(baseIdentityId);
```

Comment:
This is the only change I can see (apart from express `app.listen` method and a lot of prettier/ts fixes). 
Looks good to me!